dir.props

```diff
@@ -87,10 +72,10 @@
   <PropertyGroup>
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
-    <CoreFxExpectedPrerelease>beta-24322-03</CoreFxExpectedPrerelease>
-    <CoreClrExpectedPrerelease>beta-24322-03</CoreClrExpectedPrerelease>
-    <ExternalExpectedPrerelease>beta-24322-00</ExternalExpectedPrerelease>
-    <WCFExpectedPrerelease>beta-24329-01</WCFExpectedPrerelease>
+    <CoreFxExpectedPrerelease>beta-24431-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>beta-24431-02</CoreClrExpectedPrerelease>
```

Comment:
Just double-checking that we expect the difference between `CoreClrExpectedPrerelease` and the others